canon/CHANGELOG.md

```diff
@@ -0,0 +1,84 @@
+# 0.6.0
```

Comment:
As mentioned in issue #79, you should follow [this changelog format](https://keepachangelog.com/en/1.0.0/). As also mentioned in #79, an example of our changelog following this format can be found in [dusk-abi](https://github.com/dusk-network/dusk-abi/blob/master/CHANGELOG.md).

Spefically, what is missing here is:
```
# Changelog
All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased]
```

Notice that the version `0.6.0` is not released yet, that means, all  the changes are under `Unreleased` section and there is no `0.6.0` section.
For the same reason we should not bumped the version of canonical yet (so we should in a separate commit, not related to this PR, put it back to `0.5.3`.

Notice also the additional level for the heading (since we start with `Changelog`)